app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -147,7 +154,7 @@ fun SurveyDetailScreen(
                 .semantics { contentDescription = startSurveyDescription }
         )
         NextQuestionButton(
-            showButton = currentPage > 0,
+            showButton = currentPage > 0 && !isLastPage,
```

Comment:
How about reusing `isStartPage` too? 😄 
```suggestion
            showButton = !isStartPage && !isLastPage,
```